sphinx/transforms/i18n.py

```diff
@@ -247,13 +247,24 @@ def is_named_target(node):
             if isinstance(node, LITERAL_TYPE_NODES):
                 msgstr = '::\n\n' + indent(msgstr, ' ' * 3)
 
+            # Structural Subelements phase1
+            # There is a possibility that only the title node is created.
+            # see: http://docutils.sourceforge.net/docs/ref/doctree.html#structural-subelements
+            if isinstance(node, nodes.title):
+                # This generates: <section ...><title>msgstr</title></section>
+                msgstr = msgstr + '\n' + '-' * len(msgstr) * 2
```

Comment:
I don't think this code is good. I should refactor this `sphinx.transform.i18n` module later.